source/actions3/actions/AMDetectorTriggerAction.cpp

```diff
@@ -61,16 +61,24 @@ void AMDetectorTriggerAction::startImplementation(){
 
 	if(!detector_) {
 		AMErrorMon::alert(this,
-						  AMDETECTORTRIGGERACTION_NO_VALID_DETECTOR,
-						  QString("There was an error trigger the detector '%1', because the detector was not found. Please report this problem to the Acquaman developers.").arg(detectorTriggerInfo()->name()));
+				  AMDETECTORTRIGGERACTION_NO_VALID_DETECTOR,
+				  QString("There was an error trigger the detector '%1', because the detector was not found. Please report this problem to the Acquaman developers.").arg(detectorTriggerInfo()->detectorInfo()->name()));
+		setFailed();
```

Comment:
It did, it's just the way git has ordered the changes here. The old line is on 66 / 74 below